README.adoc

```diff
@@ -86,27 +57,27 @@ include::{common-includes}/gitclone.adoc[]
 
 === Try what you'll build
 
-The `finish` directory in the root of this guide contains the finished fault tolerance implementation
+The `finish` directory in the root of this guide contains the finished implementation
 for the application. You can give it a try before you proceed with building your own.
 
-To try out the application, first navigate to the `finish` directory. Then, execute the following
-Maven goals to build the application and run it inside Open Liberty:
+To try out the application, navigate to the `finish` directory and run the following command:
 
 ```
-cd finish
 mvn clean install liberty:start-server
 
 ```
 
-Point your browser to the `http://localhost:9080/inventory/systems/localhost` URL. You see a result
-in JSON format with the system properties of your local JVM. When you visit this URL, some of these system
+Point your browser to the `http://localhost:9080/inventory/systems/localhost` URL, which accesses the
+`inventory` service with a `localhost` host name. You see the system properties for this host.
+When you visit this URL, some of these system
 properties, such as the OS name and user name are automatically stored in the inventory.
```

Comment:
We should add a comma after "user name" in line 73.

Reply:
Fixed